src/id_token_jws.erl

```diff
@@ -73,11 +73,11 @@ generate_key_for(Alg, Options) ->
           _ -> generate_rsa_key(Alg, Options)
         end,
   Jwk0 = jose_jwk:from_key(Key),
-  Jwk = Jwk0#jose_jwk{fields = #{<<"kid">> => kid(Options),
+  JWK = Jwk0#jose_jwk{fields = #{<<"kid">> => kid(Options),
```

Comment:
So this was needed for the `consistent_variable_casing` rule? Looks a bit off with `Jwk0` and `JWK`.

Reply:
I can "fix" that. What the rule check is "lowercase comparison" of variables in the whole file. `JWk`, `JWK`, `JwK`; it doesn't care about the `0` (oh, how I miss Elixir's `|` operator)…

Reply:
I'll move all `Jw`... wait a min.

Reply:
My point is that it should also complain about `Jwk0` so that we'd have to change it `JWK0`, right?

Reply:
We can still just do that though.

Reply:
But now I see. With that `0` it's a totally different variable name according to Elvis. Not really how we see it though.